src/cairoWriter/writers/cairoAssertWriter.ts

```diff
@@ -6,6 +6,6 @@ export class CairoAssertWriter extends CairoASTNodeWriter {
   writeInner(node: CairoAssert, writer: ASTWriter): SrcDesc {
     const expression = writer.write(node.vExpression);
     const message = node.assertMessage ?? 'Assertion error';
-    return [`assert( ${expression}, "${message}" );`];
+    return [`assert( ${expression}, '${message}' )`];
```

Comment:
Shall we remove space after `(` and before `)`?

Reply:
Yes, I think it is better that way

Reply:
Yes, they can be removed